config/analytics.yml

```diff
@@ -261,6 +279,8 @@ shared:
     - id
     - message
     - updated_at
+    - user_id
```

Comment:
Should we hash?

Reply:
These are vendor/provider ids so should be ok I think